Dockerfile

```diff
@@ -13,10 +11,23 @@ RUN apt update && apt install --no-install-recommends -y \
   libssl-dev \
   pkg-config \
   postgresql \
+  netcat \
+  curl \
+  && curl -o /wait-for https://raw.githubusercontent.com/Eficode/wait-for/master/wait-for \
+  && chmod +x /wait-for \
+  && apt-get remove -y curl \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /oss-dashboard
```

Comment:
With WORKDIR used here, you probably don't want the extra `/oss-dashboard`s in the subsequent COPY commands. You'll end up with `/oss-dashboard/oss-dashboard/*`.

Reply:
The subsequent commands are using absolute paths which makes them independent of the current working directory. So I would say this is a non-issue, or did something fail in your environment?

Reply:
P.S.: I've trained myself to keep COPY/ADD commands using absolute paths, because it's sometimes hard to grasp which the current workdir is. Also, it makes refactorings more stable. Yet, I'm aware that when changing the base working directory you'll also have to change all absolute paths. That's a tradeoff... if you prefer shorter paths, I can certainly change this patch to use relative paths.

Reply:
Got it. I missed that fact.